### ISSUE TITLE
Openbangla is not working on NIXOS.

### ISSUE THREAD

**Issue by nahian-adnan** [OpenBangla/OpenBangla-Keyboard#346]
Hey, openbangla does not function properly on NIXOS. This package has been made to function, but it does not function correctly with ibus or fcitx5.

**Comment by asifakonjee** [comment]
You can use ibus and openbangla keyboardin any x11 session. If you are using hyprland then you have to use a script to change language and also have to use a cuatom module in waybar as ibus does not show in system tray in wayland. But ibus and openbangla works perfectly in NixOS.

Sadly, the better option (fcitx5 and openbangla) does not work.

**Comment by asifakonjee** [comment]
@nahian-adnan... OpenBangla Keyboard is not working with fcitx5 but it is working perfectly with ibus in NixOS.

**Comment by nahian-adnan** [comment]
> @nahian-adnan... OpenBangla Keyboard is not working with fcitx5 but it is working perfectly with ibus in NixOS.

yes, and i don't need it anymore. btw i think someone should add fcitx5 support for it.

**Comment by Linux-DADDY** [comment]
@asifakonjee well, i hope one day it will get fixed.

**Comment by asifakonjee** [comment]
@Linux-DADDY ... Solved already as per my knowledge. i opened several issues in fcitx5 and as well as nixpkgs github and they finally solved the problem.

**Comment by Linux-DADDY** [comment]
@asifakonjee, that's a great news, will look at it today.

**Comment by Linux-DADDY** [comment]
@asifakonjee , Hello again, So i have tried to make it work, but the problem i have found is it is not working at all, maybe i have did it wrong or maybe i didn't able to make it work. What i did is, I installed the fcitx5-openbangla-keyboard then i search openbangla found there is no openbangla option, i am not familiar with fxitx5 qt or gtk. Correct me if i am wrong here.

**Comment by asifakonjee** [comment]
@Linux-DADDY ... Sorry for late reply. You also have to install fcitx5 and fcitx5-with-addons.

**Comment by asifakonjee** [comment]
@Linux-DADDY ... কাজ করছে কোন সমস্যা ছাড়াই।

![NixOS ](https://github.com/user-attachments/assets/26eaa02c-2b2e-4be9-a325-b8b39e55499c)
